core/controllers/custom_landing_pages.py

```diff
@@ -30,12 +28,11 @@ class FractionLandingPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        version_id = self.request.get('v')
+        viewerType = self.request.get('viewerType')
```

Comment:
viewer_type for python vars, ditto below

Reply:
Done